app/controllers/spree/admin/payments_controller.rb

```diff
@@ -9,6 +9,7 @@ class PaymentsController < Spree::Admin::BaseController
       before_action :load_payment, only: [:fire, :show]
       before_action :load_data
       before_action :can_transition_to_payment
+      before_action :ensure_sufficient_stock_lines
```

Comment:
Do we want to run this on al of this controller's actions? Is it really needed on index, for example? I'm wondering if this should be scoped to specific actions, like:
```ruby
      before_action :ensure_sufficient_stock_lines, only: [:create, :update]
```
:thinking: